lighty-modules/lighty-openflow-sb/src/test/java/io/lighty/modules/southbound/openflow/tests/OpenflowSouthboundPluginTest.java

```diff
@@ -8,11 +8,11 @@
 package io.lighty.modules.southbound.openflow.tests;
 
 import org.testng.Assert;
-import org.testng.annotations.Test;
 
 public class OpenflowSouthboundPluginTest extends OpenflowSouthboundPluginTestBase {
 
-    @Test(timeOut = 60_000)
+//    @Test(timeOut = 60_000)
+//    TODO: Allow when will be released OFP at version 0.12.0 or higher and serviceutils at 0.7.0 or higher
```

Comment:
With OFP snapshot versions this should be activated again

Reply:
done